runelite-client/src/main/java/net/runelite/client/plugins/experiencedrop/XpDropConfig.java

```diff
@@ -87,4 +87,14 @@ default int fakeXpDropDelay()
 		return 0;
 	}
 
+	@ConfigItem(
+		keyName = "disableFakeXpDrops",
+		name = "Disable fake xp drops",
+		description = "Disables fake exp drops",
```

Comment:
I feel the extra explanation would be more useful to include than not.

```suggestion
		description = "Disables fake exp drops (such as post-200m or ironman pvp exp)",
```